packages/password/html-testing/generate.js

```diff
@@ -0,0 +1,48 @@
+const {readFileSync, writeFileSync} = require('fs')
```

Comment:
Probably worth adding a comment block to explain what this does and how to use it. Love the idea, though.